apps/system/js/nfc_handover_manager.js

```diff
@@ -201,32 +161,8 @@
     },
 
     '_handle_bluetooth-enabled': function() {
-      if (typeof(window.navigator.mozBluetooth.onattributechanged) !==
-        'undefined') {
-        this._debug('Bluetooth APIv2 does not support NFC yet');
-        return;
-      }
-      var self = this;
-      self._debug('bluetooth-enabled');
-      var req = self.bluetooth.getDefaultAdapter();
-      req.onsuccess = function bt_getAdapterSuccess() {
-        self.settingsNotified = false;
-        self.defaultAdapter = req.result;
-        self._debug('MAC address: ' + self.defaultAdapter.address);
-        self._debug('MAC name: ' + self.defaultAdapter.name);
-        /*
-         * Call all actions that have queued up while Bluetooth
-         * was turned on.
-         */
-        for (var i = 0; i < self.actionQueue.length; i++) {
-          var action = self.actionQueue[i];
-          action.callback.apply(self, action.args);
-        }
-        self.actionQueue = [];
-      };
-      req.onerror = function bt_getAdapterError() {
-        self._logVisibly('event listner: Failed to get bluetooth adater');
-      };
+      this._debug('bluetooth-enabled');
+      this._processQueue();
```

Comment:
nit: I will say if this won't be called somewhere else, it's not needed to have a new function